spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  it "is an instance of the facility factory class" do
+    factory = FacilityFactory.new
+    expect(factory).to be_an_instance_of(FacilityFactory)
+  end
+
+  it "can create facility objects from an external database" do
+    @co_dmv_office_locations = DmvDataService.new.co_dmv_office_locations
+    @ny_dmv_office_locations = DmvDataService.new.ny_dmv_office_locations
+    @mo_dmv_office_locations = DmvDataService.new.mo_dmv_office_locations
+    factory = FacilityFactory.new
+
+    output1 = factory.create_facility(@co_dmv_office_locations)
+    output2 = factory.create_facility(@ny_dmv_office_locations)
+    output3 = factory.create_facility(@mo_dmv_office_locations)
+
+    expect(output1).to be_an(Array)
+    expect(output1.first).to be_a(Facility)
+
```

Comment:
To dig further into testing here, we could test that output1.first.address is a String, or check that it's not nil, and we could do that for all of the attributes that we expect to be set after this method runs.